### DISCUSSION TITLE
upgrade from 5.0.0 to 6.0.0

### DISCUSSION THREAD

**Discussion by Venkatesh-Dayani23** [Sunbird-Ed/Community#704] [General Discussion]
Hi Team,
Currently we are using the Sunbird 5.0.0 and we are planning to upgrade to 6.0.0 directly which means we will do all the upgrades included in 5.1.0, 5.2.0 and 6.0.0 at once. Can we get any suggestions on this please?

cc: @raghupathiguduri

**Comment by gandham-santhosh** [comment]
@Venkatesh-Dayani23 we recommend to upgrade to each consequent release to upgrade from 5.0.0 to 6.0.0 
5.0.0 -> 5.1.0 -> 5.2.0 -> 6.0.0

**Reply by Venkatesh-Dayani23** [reply]
ok @gandham-santhosh  Thanks.
